README.md

```diff
@@ -43,10 +43,11 @@ If you want to contribute financially and help us keep the application free and
 ## Run locally
 
 1. Clone the repository (or fork it if you intend to contribute)
-2. `npm install`
+2. Run `npm install` to install dependencies
 3. Start a PostgreSQL server. You can run `./scripts/start-local-db.sh` if you don’t have a server already.
 4. Copy the file `.env.example` as `.env`
-5. `npm run dev`
+5. Run `npm run postinstall` to apply database migrations and update Prisma Client
```

Comment:
`npm run postinstall` is unnecessary as it is run by `npm install` automatically.

Reply:
We may need to adjust the instructions or order of operations, as `postinstall` fails when the PostgreSQL server is not running.

Reply:
You’re absolutely right! Maybe put the instruction about the DB server before anything else then 😉

Reply:
I have updated the instructions.